liferay-workspace/tests/src/test/java/com/liferay/blade/tests/BladeCLI.java

```diff
@@ -101,7 +103,11 @@ public static String getLatestBladeCLIJar() throws Exception {
 			File[] files = repo.get( "com.liferay.blade.cli", "[1,2)" );
 			File cliJar = files[0];
 
-			bladeJar = cliJar;
+			File newCliJar = new File(repoPath + "/" + cliJar.getName());
```

Comment:
No need to copy to a new location.  You can invoke it in the old location, we just need to encode the execution parameter in quotes.

so earlier in this file

```
command.add("java");
command.add("-jar");
command.add(bladeCLIJarPath);
```

We can just put bladeCliJarPath in quotes.